core/server/api/canary/newsletters.js

```diff
@@ -11,15 +11,15 @@ module.exports = {
             'order',
             'page'
         ],
-        permissions: () => true, //FIXME: move to database permissions
```

Comment:
IMO This change should be part of the other PR, I don't like that we've got commits which explicitly disable permissions on our endpoints. Not huge deal but 🤷 I would just merge this followed by the API implementation

Reply:
I'm gonna merge the controller with `permissions: true` and with the tests commented out, otherwise I would have to mess with the commits too much. I agree it's better to not leave this endpoint open at any time 👍